doc/man/8/ceph.rst

```diff
@@ -874,6 +874,18 @@ Usage::
 
 	ceph osd out <ids> [<ids>...]
 
+Subcommand ``ok-to-stop`` checks whether the list of OSD(s) can be
+stopped without reducing immediately data availability.  That is, all
```

Comment:
s/reducing immediately data availability/immediately making data unavailable/ is a much more accurate description of what it's doing.